content/SCALE/GettingStarted/Configure/SetUpStorageScale.md

```diff
@@ -68,8 +64,38 @@ Organize the pool with as many datasets or zvols you need according to your acce
 
 See [Adding or Managing Datasets]({{< relref "DatasetsSCALE.md" >}}) for more information on configuring datasets, or [Adding or Managing Zvols]({{< relref "AddManageZvols.md" >}}) for more information on zvols.
 
-## Taking the Next Step
+#### Adding a Dataset and Share
+TrueNAS provides the option to create the dataset and share at the same time.
+The **Add Dataset** screen allows you to create the new dataset and use a preset to configure an SMB, NFS, or multi-mode share.
+The **Shares** screen also provides options to add an SMB or NFS share and create the dataset at the same time.
+
+{{< hint type="warning" title="Sharing the Root Dataset" >}}
+Do not set up sharing on the root dataset!
+Creating a share that uses the root dataset causes all sorts of problems with permissions, and is not a best practice.
```

Comment:
This is a bit colloquial. How about "causes various permissions complications" or something along that line?

Reply:
The point of tech writing is to communicate with everyone in a simple, uncomplicated way. There is nothing wrong with being "colloquial". Simple, direct, and easy to translate is the objective.